src/target/riscv/riscv-013.c

```diff
@@ -549,11 +549,11 @@ static int dmi_op(struct target *target, uint32_t *data_in, int dmi_op,
 					status);
 			return ERROR_FAIL;
 		}
-		if (time(NULL) - start > riscv_command_timeout_sec) {
+		if (time(NULL) - start > timeout_sec) {
 			LOG_ERROR("dmi.op is still busy after %d seconds. The target is "
 					"either really slow or broken. You could increase the "
 					"timeout with riscv set_command_timeout_sec.",
```

Comment:
is this message true in the case that this fails when you use a different comment?

Reply:
Just a thought, could you return an ERROR_TIMEOUT and put the message in the caller?